amixr/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "display_name": "Amixr",
+  "maintainer": "ildar@amixr.io",
+  "manifest_version": "1.0.0",
+  "name": "amixr",
+  "metric_prefix": "amixr.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Developer-friendly Alert Management with brilliant Slack integration",
+  "guid": "56e3b585-27b4-4a04-b528-9c1adfecf387",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Amixr Integration",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "amixr",
+  "assets": {
+    "dashboards": {},
+    "saved_views": {},
+    "monitors": {},
+    "service_checks": "assets/service_checks.json"
```

Comment:
It looks like the CI is failing because it's expecting this `service_checks.json` file. 

To pass the validation you can create this file under the `assets/` directory with `[]`.